### PR TITLE
Add block for 6.8.2 to helper library

### PR DESCRIPTION
**Pull request by alzayatm**
## Description
Add a block for 6.8.2 to helper library -- includes SHA-256 checksums

### Issues Resolved
Addition of new confluence version (6.8.2)

### Check List
- [X] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [X] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures Atlassian Confluence'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.5.0'
+version '2.5.1'
```
Comment (anuriq) [inline metadata.rb]: version is usually bumped after merge, no?

**Review by damacus — CHANGES_REQUESTED** [review]
Please don't bump the version without either asking.

**Comment by alzayatm** [comment]
I did bump the version. Are you expecting a major or a minor version bump?

**Comment by alzayatm** [comment]
What is the status on this?

**Comment by anuriq** [comment]
> What is the status on this?

Hi @alzayatm 
@damacus  requested a change in the PR. Version of the cookbook is bumped after merge, before cookbook release. There may be a lot of changes during a cookbook release, if every PR bumps version, we will have inconsistent chain of versions.
Thank you.

**Comment by alzayatm** [comment]
> > What is the status on this?
> 
> Hi @alzayatm
> @damacus requested a change in the PR. Version of the cookbook is bumped after merge, before cookbook release. There may be a lot of changes during a cookbook release, if every PR bumps version, we will have inconsistent chain of versions.
> Thank you.

Understood. I changed the version back to 2.5.0. Can you guys accept the pull request now?

**Comment by anuriq** [comment]
LGTM
@damacus

**Comment by alzayatm** [comment]
Can we please approve this?

**Comment by alzayatm** [comment]
Following up again.

**Pull request merged**